app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -274,6 +274,14 @@ i.icon-xl {
   font-size: 3rem;
 }
 
+.tooltip-info {
+  color: var(--stanford-digital-blue);
+}
+
+.tooltip-shift-right {
+  left: 15px !important;
```

Comment:
I tweaked a great many things to get the Bootstrap tooltip to render directly atop the icon element, but it kept wanting to be 15px to the left. This shifts it back where we want it, I think. I'm open to alternatives, of course.